packages/vuetify/src/mixins/overlayable.ts

```diff
@@ -211,7 +211,19 @@ export default Vue.extend<Vue & Toggleable & Stackable & options>().extend({
       if (this.$vuetify.breakpoint.smAndDown) {
         document.documentElement!.classList.add('overflow-y-hidden')
       } else {
-        window.addEventListener('wheel', this.scrollListener as EventHandlerNonNull, { passive: false })
+        // IE11 does not support passive
```

Comment:
Move this to util/helpers, we use passive in other places too.

Reply:
Do I also need to find all the other passive usages and update them to use this from helpers after I move it?

Reply:
Yes please